packages/IMAPClient-Core.package/ICFolder.class/instance/fetchLatestEmails..st

```diff
@@ -1,12 +1,12 @@
-email access
-latestEmails: count
+action - update emails
+fetchLatestEmails: aCount
```

Comment:
I think aNumber sounds better than aCount.
Also this method could be renamed to something like fetchANumberOfEmailsWith: aNumber